src/styles/pages/_live.scss

```diff
@@ -86,15 +86,31 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-  margin: 64px auto;
+  margin: (64px - $WEB_HEADER_HEIGHT - 48px) auto 64px;
```

Comment:
Where does this 48 come from?

Reply:
It's just to push the content 48px past the header. I confess it's a magic number.